aksel.nav.no/website/app/dev/_ui/Highlight.tsx

```diff
@@ -0,0 +1,133 @@
+import cl from "clsx";
+import Image from "next/legacy/image";
+import NextLink from "next/link";
+import { BodyLong, BodyShort, HStack, Heading, Link } from "@navikt/ds-react";
+import { umamiTrack } from "@/app/_ui/umami/Umami.track";
+import { useFormatedDate } from "@/hooks/useFormatedDate";
+import { urlFor } from "@/sanity/interface";
+import { getAuthors, getImage } from "@/utils";
+import { ArticleT } from "./Card";
+import { BetaTag, Tag } from "./Tag";
+import styles from "./landingpage.module.css";
+
+export const Highlight = ({
+  article,
+  compact,
+}: {
+  article: ArticleT;
+  compact: boolean;
+}) => {
+  const showFooter = ["aksel_artikkel", "aksel_blogg"].includes(article._type);
+  const useStatusImage =
+    ["ds_artikkel", "komponent_artikkel"].includes(article._type) &&
+    article.status?.bilde;
+
+  const date = useFormatedDate(article?.publishedAt ?? article._createdAt);
+
+  const imageUrl = urlFor(article.status?.bilde)
+    ?.quality(100)
+    .auto("format")
+    .url();
+
+  const imageBlurUrl = urlFor(article.status?.bilde)
+    ?.width(24)
+    .height(24)
+    .blur(10)
+    .url();
+
+  const seoImageUrl = urlFor(article?.seo?.image)
+    ?.quality(100)
+    .auto("format")
+    .url();
+
+  const seoImageBlurUrl = urlFor(article?.seo?.image)
+    ?.width(24)
+    .height(24)
+    .blur(10)
+    .url();
+
+  return (
+    <section
+      aria-label={`Fremhevet artikkel: ${article?.heading}`}
+      className={cl({
+        [`${styles.highlightSectionCompact}`]: compact,
+        [`${styles.highlightSection}`]: !compact,
+      })}
+    >
```

Comment:
```suggestion
    <HGrid
      aria-label={`Fremhevet artikkel: ${article?.heading}`}
      align="start"
      gap={compact ? "space-32" : "space-24"}
    >
```
This is a 1-1 from the CSS now, but why does it have more space when "compact"?

Reply:
Can remove the `highlightSectionCompact` and `highlightSection` classes.

Reply:
maybe i swapped them during conversion from tw 🤔, will do a final pass with designer on this as well

Reply:
turns out, the logic is good, but the name is a bit misleading. renamed from `compact` -> `singleHighlightedArticle`, having a single thing is in a way "more compact" but not really correct verbiage 😅. it makes sense that when you have a single thing it might get a little more space around it though (which is the inverse of "compact")